src/components/Nav/Nav.Props.ts

```diff
@@ -78,17 +78,21 @@ export interface INavLink {
   isExpanded?: boolean;
 
   /**
-   * Any additional properties to apply to the rendered links.
+   * (Optional) Aria label for nav link
    */
-  [propertyName: string]: any;
+  ariaLabel?: string;
 
   /**
-   * (Optional) Aria label for nav link
+   * (Optional) Meta info for the link, does not involving rendering.
    */
-  ariaLabel?: string;
+  key?: string;
 
   /**
-   * (Optional) Title for nav link
+   * (Optional) title for tooltip or description
    */
   title?: string;
+  /**
+   * Any additional properties to apply to the rendered links.
+   */
+  [propertyName: string]: any;
```

Comment:
I thought this field [propertyName: string]: any always there ?  Unless someone just removed it?  I have not sync up to the latest?  I don't need this and I think this was originally added from day one to mimic the ILeftNavItem customListProperties.
I can remove it if you think that is right thing to do.